modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/sequence/StepsControlButtons.scala

```diff
@@ -32,10 +34,10 @@ object StepsControlButtons {
     Callback(SeqexecCircuit.dispatch(RequestAbort(id, stepId)))
 
   def handleStop(id: SequenceId, stepId: Int): ScalazReact.ReactST[CallbackTo, State, Unit] =
-    ST.retM(requestStop(id, stepId)) >> ST.mod(_.copy(stopRequested = true)).liftCB
+    ST.retM(requestStop(id, stepId)) >> ST.mod(_.copy(stopRequested = true, abortRequested = true)).liftCB
```

Comment:
Why are both true?

Reply:
This is local state, to disable the buttons until a response is heard from the backend. Both true means that `Stop` and `Abort` buttons will be disabled until the response of either is received

Basically it means you can't do `Abort` while `Stopping` and viceversa

Reply:
Ah, ok.